securedrop/source_templates/index.html

```diff
@@ -41,72 +39,47 @@ <h2 id="security-level-heading" hidden>{{ gettext('How to change your security l
     <p>
       {{ gettext('<a href="/" aria-label="Follow these instructions, then refresh this page">Refresh this page</a>, and you\'re done!') }}
     </p>
-
   </dialog>
-  <div id="use-tor-browser" class="warning" role="alert">
+  <div id="browser-tb" class="warning" role="alert">
     {{ gettext('<strong>It is recommended to use Tor Browser to access SecureDrop:</strong> <a class="recommend-tor" href="{tor_browser_url}">Learn how to install it</a>, or ignore this warning to continue.').format(tor_browser_url=url_for('info.recommend_tor_browser')) }}
```

Comment:
Uses default `a` link colors instead of original white (for both visited and unvisited). This doesn't appear to be covered in design mockups, but the combo of blue and purple is pretty low-contrast, the original white color works better imo.

Reply:
Hmm, what browser does this happen with for you? Both Chromium and Firefox show the link here as white and underlined for me

Reply:
Yikes, Firefox 78esr apparently, will update the test environment  and try it again.

Reply:
Yupp, fine in 91esr, my bad.